modules/nw-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+// * networking/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-aws-load-balancer-operator_{context}"]
+= AWS Load Balancer Operator
+
+The AWS Load Balancer Operator can tag the public subnets if the `kubernetes.io/role/elb` tag is missing. Also, the AWS Load Balancer Operator detects the following from the underlying AWS cloud:
+
+* The ID of the virtual private cloud (VPC) on which the cluster hosting the operator is deployed in.
+* Public and private subnets of the discovered VPC.
+
+.Prerequisites
+
+* You must have the AWS credentials secret. The credentials are used to provide subnet tagging and VPC discovery.
+
+.Procedure
+
+. You can deploy the AWS Load Balancer Operator on demand from the OperatorHub, by creating a `Subscription` object:
++
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator get sub aws-load-balancer-operator -o yaml | yq '.status.installplan.name'
```

Comment:
Question on this - I'm not familiar with `yq`, is that like `jq`, where you users are going to have to install this before this command will work? 

Just thinking that this might be along the same lines of something that we want to avoid (see "Do not use jq in commands..." in the doc guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)

Can you follow up with your SMEs to see if yq in this command should be replaced with an equivalent command that uses jsonpath instead?

@kalexand-rh Checking whether we should add 'yq' to the guidelines for things to avoid, since users would have to install it before the command would work. wdyt?

Reply:
If it's not a standard package in all supported versions in RHEL, then yes, it's the same as `jq`.

Reply:
@alebedev87 Do we have to replace `yq` with an equivalent command that uses JSON path instead?

Reply:
It doesn't really matter `yq` or `jq`. We can come out with a command which doesn't use any other utilities but `oc` if that's what's recommended.
But if `jq` is fine, I'm good with it too, it's just that the `oc` command has to output in json then: `-o json`.

Reply:
This command will give the same result without using any additional tools:
```
oc -n aws-load-balancer-operator get sub aws-load-balancer-operator --template='{{.status.installplan.name}}{{"\n"}}'
```

Reply:
Super...thanks

Addressed